sql/database.sql

```diff
@@ -2188,8 +2196,12 @@ CREATE TABLE `form_vitals` (
   `oxygen_saturation` float(5,2) default '0.00',
   `oxygen_flow_rate` float(5,2) default '0.00',
   `external_id` VARCHAR(20) DEFAULT NULL,
+  `ped_weight_height` float(4,1) default '0.0',
+  `ped_bmi` float(4,1) default '0.0',
+  `ped_head_circ` float(4,1) default '0.0',
```

Comment:
not sure if this is a meaningful difference, but in the upgrade sql and table sql, default is 0.00, but here is 0.0

Reply:
I changed the values to be 0.00.